.github/scripts/check-grpc-coverage.py

```
@@ -0,0 +1,304 @@
+        if query_name not in EXCLUDED_QUERIES:
+            queries.append(query_name)
+    
+    return queries
+
+
+def check_query_implementation(query_name, sdk_path):
+    """Check if a query is implemented in the SDK."""
+    patterns = QUERY_MAPPINGS.get(query_name, [query_name])
+    
+    # Search for any of the patterns in the SDK code
+    for root, dirs, files in os.walk(sdk_path):
```

Ruff (0.11.9)

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)